public/index.css

```diff
@@ -209,7 +209,7 @@ html {
 }
 
 .ms-Button-icon:not(:disabled):hover {
-  color: var(--color-themeDark) !important;
+  color:rgb(252, 167, 9) !important;
```

Comment:
Please add this as a color to the color table rather than refer to RGB values directly.  Thanks!